site/profile/templates/cvmfs/ref.mugqic.conf.epp

```diff
@@ -0,0 +1,4 @@
+CVMFS_REPOSITORIES=ref.mugqic
+CVMFS_ALIEN_CACHE="/scratch/cvmfs_alien_cache/ref.mugqic"
```

Comment:
Is this directory automatically created by MC with the right ownership and privileges?
chgrp cvmfs and chmod 770  should be sufficient.

Reply:
it is done in magic castle in the cloud init step:  https://github.com/ComputeCanada/magic_castle/pull/118/files